piptools/resolver.py

```diff
@@ -389,7 +404,9 @@ def _iter_dependencies(self, ireq):
                 dependency_string, constraint=ireq.constraint, comes_from=ireq
             )
 
-    def reverse_dependencies(self, ireqs):
+    def reverse_dependencies(
+        self, ireqs: Set[InstallRequirement]
```

Comment:
Only used for iteraration.

```suggestion
        self, ireqs: Iterable[InstallRequirement]
```

Reply:
Right, thanks! Addressed in 386c8e07c054176758ed65a937c9fbc9dad0c1d2.